src/pages/learn.js

```diff
@@ -1,25 +1,48 @@
-import React, { useContext } from "react"
+import React, { useContext, useEffect } from "react"
 import Navbar from "../components/navbar"
 import IDE from "../components/ide"
 import Term from "../components/term"
 import Video from "../components/video"
-import { HeartwoodDispatchContext } from "../state/HeartwoodContextProvider"
+import { HeartwoodStateContext, HeartwoodDispatchContext } from "../state/HeartwoodContextProvider"
```

Comment:
Codacy found an issue: ['HeartwoodDispatchContext' is defined but never used.](https://app.codacy.com/gh/treetoplearning/heartwood/pullRequest?prid=5837031)